### PR TITLE
Added missing metrics in the docs

### PR DESCRIPTION
**Pull request by borisvuchkov**
Added missing metrics in the docs:
- network growth
- network profit loss
- sentiment
- social dominance
- social volume 
- transaction volume 

## Checklist

- [ ] My article available in Navigation sidebar and in root article of selected category ([how to do it in README](https://github.com/santiment/academy#add-an-article-into-navigation-sidebar))
- [ ]	My article have [metadata](https://github.com/santiment/academy#metadata) (title, description, date when updated and author)
- [ ]	All added/updated links in article are exist, images shows correctly

### CODE REVIEW

File: src/docs/metrics/development-activity/github-activity/index.md [hunk 1]
```diff
@@ -149,3 +149,14 @@ To check what assets are part of the ecosystem and what are their github links:
...
 
+## Full list of metrics
+
+The full list of GitHub Activity metrics is:
+
+<Details>
+
+<Summary>Open Metrics List</Summary>
+
+- github_activity_change_30d
```
Comment (alexander-gr) [inline src/docs/metrics/development-activity/github-activity/index.md:160]: Api broken

File: src/docs/metrics/sentiment-metrics/index.md [hunk 2]
```diff
@@ -82,3 +82,55 @@ out neutral, spam, or irrelevant messages, focusing on the most impactful data.
...
+- sentiment_volume_consumed_reddit
+- sentiment_volume_consumed_telegram
+- sentiment_volume_consumed_twitter
+- sentiment_volume_consumed_twitter_crypto
+- sentiment_volume_consumed_twitter_news
+- sentiment_volume_consumed_twitter_nft
+- sentiment_volume_consumed_youtube_videos
+- sentiment_volume_consumed_total_change_1d
+- sentiment_volume_consumed_total_change_30d
+- sentiment_volume_consumed_total_change_7d
```
Comment (alexander-gr) [inline src/docs/metrics/sentiment-metrics/index.md:134]: `_consumed_` metrics are not described in the article

File: src/docs/metrics/development-activity/github-activity/index.md (hunk 1 above)
Comment (alexander-gr) [reply src/docs/metrics/development-activity/github-activity/index.md:160]: Let's add this example:
```
{
  getMetric(metric: "github_activity_change_30d") {
    timeseriesData(
      selector: {slug: "ethereum"}
      from: "2024-01-01T00:00:00Z"
      to: "2024-01-07T00:00:00Z"
      interval: "1d"
    ) {
      datetime
      value
    }
  }
}
```

File: src/docs/metrics/bitmex-derivatives/perpetual-contract-funding-rate/index.md [hunk 3]
```diff
@@ -77,3 +77,17 @@ Available under the `bitmex_perpetual_funding_rate` name.
...
+
+The full list of Bitmex Perpetual Funding Rate metrics is:
+
+<Details>
+
+<Summary>Open Metrics List</Summary>
+
+- bitmex_perpetual_funding_rate_change_1d
+- bitmex_perpetual_funding_rate_change_30d
+- bitmex_perpetual_funding_rate_change_7d
```
Comment (IvanIvanoff) [inline src/docs/metrics/bitmex-derivatives/perpetual-contract-funding-rate/index.md:91]: not a full list, only the change metrics are here.

File: src/docs/metrics/historical-balance/index.md [hunk 4]
```diff
@@ -68,3 +68,16 @@ Query variables:
...
+The full list of Historical Balance metrics is:
+
+<Details>
+
+<Summary>Open Metrics List</Summary>
+
+- historical_balance
+- historical_balance_changes
+
+</Details>
```
Comment (IvanIvanoff) [inline src/docs/metrics/historical-balance/index.md:83]: This is not a metric, it should not be listed here.

File: src/docs/metrics/marketcap/index.md [hunk 5]
```diff
@@ -74,3 +74,17 @@ Available assets for [marketcap_usd](https://api.santiment.net/graphiql?query=%7
...
+
+The full list of Price USD metrics is:
+
+<Details>
+
+<Summary>Open Metrics List</Summary>
+
+- marketcap_usd_change_1d
+- marketcap_usd_change_30d
+- marketcap_usd_change_7d
```
Comment (IvanIvanoff) [inline src/docs/metrics/marketcap/index.md:88]: not a full list, only the change metrics are here.

File: src/docs/metrics/network-growth/index.md [hunk 6]
```diff
@@ -95,3 +95,17 @@ Available under the `network_growth` name.
...
+
+The full list of Network Growth metrics is:
+
+<Details>
+
+<Summary>Open Metrics List</Summary>
+
+- network_growth_change_1d
+- network_growth_change_30d
+- network_growth_change_7d
```
Comment (IvanIvanoff) [inline src/docs/metrics/network-growth/index.md:109]: not all metrics are listed, only the change metrics.

File: src/docs/metrics/price/index.md [hunk 7]
```diff
@@ -170,4 +170,18 @@ Amount in dollars/usdt/bitcoins
...
+
+The full list of Price USD metrics is:
+
+<Details>
+
+<Summary>Open Metrics List</Summary>
+
+- price_usd_change_1d
+- price_usd_change_30d
+- price_usd_change_7d
```
Comment (IvanIvanoff) [inline src/docs/metrics/price/index.md:185]: not all metrics are listed, only the change metrics.

File: src/docs/metrics/social-dominance/index.md [hunk 8]
```diff
@@ -149,3 +149,48 @@ Explorer](<https://api.santiment.net/graphiql?query=%7B%0A%20%20getMetric(metric
...
+- social_dominance_total_change_7d
+- social_dominance_twitter
+- social_dominance_twitter_1h_moving_average
+- social_dominance_twitter_24h_moving_average
+- social_dominance_twitter_crypto
+- social_dominance_twitter_news
+- social_dominance_twitter_nft
+- social_dominance_youtube_videos
+- social_dominance_youtube_videos_1h_moving_average
+- social_dominance_youtube_videos_24h_moving_average
```
Comment (IvanIvanoff) [inline src/docs/metrics/social-dominance/index.md:194]: We should not list `twitter_nft`, `twitter_crypto`, `twitter_news` -- they are deprecated and are not computed.

Furthermore, we split the available metrics into multiple dropdowns, for example here they can be `Metrics`, `Change Metrics`, `Moving Average` metrics, etc.

File: src/docs/metrics/transaction-volume/index.md [hunk 9]
```diff
@@ -78,3 +78,20 @@ Available under the `transaction_volume` name.
...
+<Details>
+
+<Summary>Open Metrics List</Summary>
+
+- transaction_volume_change_1d
+- transaction_volume_change_30d
+- transaction_volume_change_7d
+- transaction_volume_usd_change_1d
+- transaction_volume_usd_change_30d
+- transaction_volume_usd_change_7d
```
Comment (IvanIvanoff) [inline src/docs/metrics/transaction-volume/index.md:95]: not all metrics are listed, only the change metrics.

File: src/docs/metrics/total-supply/index.md [hunk 10]
```diff
@@ -130,3 +130,16 @@ Percent of Supply in Profit is available for  `percent_of_total_supply_in_profit
...
+
+## Full list of metrics
+
+The full list of Total Supply metrics is:
+
+<Details>
+
+<Summary>Open Metrics List</Summary>
+
+- top_holders_percent_of_total_supply
```
Comment (IvanIvanoff) [inline src/docs/metrics/total-supply/index.md:142]:
```suggestion
```
This is not a metric, but a separate API query. Let's remove it from here.

File: src/docs/metrics/trading-volume/index.md [hunk 11]
```diff
@@ -84,3 +84,16 @@ Available under `volume_usd` and `volume_usd_5m` names
...
+## Full list of metrics
+
+The full list of GitHub Activity metrics is:
+
+<Details>
+
+<Summary>Open Metrics List</Summary>
+
+- volume_usd_change_30d
+- volume_usd_change_7d
```
Comment (IvanIvanoff) [inline src/docs/metrics/trading-volume/index.md:97]:
```suggestion
- volume_usd
- volume_usd_5m
- volume_usd_change_1d
- volume_usd_change_30d
- volume_usd_change_7d
```

**Pull request merged**